### ISSUE TITLE
WebVMT: Review map block interface exposure

### ISSUE THREAD

**Issue by rjksmith** [w3c/sdw#1100]
@tidoust wrote:

The [WebVMT map definition block](http://w3c.github.io/sdw/proposals/geotagging/webvmt/#dfn-webvmt-media-definition-block) is interesting because the information it contains needs to be exposed to Web applications somehow, and there is no mechanism to do so for now. That is, even if browsers could parse and expose WebVMT cues as DataCue or as a specific VMTCue, the map information would still need to be exposed somehow. I'd raise that as an open issue at least.

**Comment by rjksmith** [comment]
The [WebVMT map block](http://w3c.github.io/sdw/proposals/geotagging/webvmt/#dfn-webvmt-map) is analogous to the [WebVTT region block](https://www.w3.org/TR/webvtt1/#webvtt-region-definition-block), as it describes the area for rendered output. WebVTT exposes regions through the [WebVTT region interface](https://www.w3.org/TR/webvtt1/#the-vttregion-interface) and I'd expect WebVMT would have an equivalent interface.

I agree that this is an omission from the current Editor's Draft and should be added.

**Comment by rjksmith** [comment]
**SDW IG Review Summary (5/12/18)**

This is an omission and the [WebVMT map block](
http://w3c.github.io/sdw/proposals/geotagging/webvmt/#dfn-webvmt-map) should be exposed.

_Conclusion_
   * Add a WebVMT map interface to expose this information and close this issue.

**Comment by rjksmith** [comment]
Updated Editor's Draft - pull request #1105:
   * Added a section on [Planned Interfaces](http://w3c.github.io/sdw/proposals/geotagging/webvmt/#plannedinterfaces) to Known Issues, including draft VMTCue and VMTMap interfaces.

Formal API definition is separate from the Data Model and Syntax sections, and so beyond the scope of the 5/12/18 review. I've added an informal draft for now, to serve as a starting point for the formal definition in due course.
